src/services/housing.js

```diff
@@ -22,6 +22,21 @@ const getHousingInfo = async () => {
   return housingInfo;
 };
 
+/**
+ * Get active apartment applications for current user
+ * @param {String} primaryUsername the student username of the person filling out the application
+ * @param {StudentProfileInfo} applicants Array of StudentProfileInfo objects
+ * @return {Promise.<String>} User's active jobs
+ */
+const saveApartmentApplication = async (primaryUsername, applicants) => {
+  let applicationDetails = {
+    PRIMARY_USERNAME: primaryUsername,
+    APPLICANT_ARRAY: applicants.map(profile => profile.AD_Username),
+  };
+  return await http.post(`housing/submit/`, applicationDetails);
```

Comment:
I think we should call the route 'save', not 'submit', but you probably were just going off of what Nick had...

Reply:
I completely agree, especially since there will actually be a 'submit' for when they are done with the application